src/main/java/ca/uhn/fhir/jpa/starter/BaseJpaRestfulServer.java

```diff
@@ -106,7 +106,7 @@ public BaseJpaRestfulServer() {
   @Override
   protected void initialize() throws ServletException {
     super.initialize();
-
+    daoConfig.setUseLegacySearchBuilder(true);
```

Comment:
When did this problem start? Has this issue been raised with the open-source (upstream) repo? I don't like the idea of relying on a legacy flag like this.

Reply:
We discussed this during our scrum call today. I'm approving it, but added a Jira ticket for a later sprint to see if the upstream has resolved it and remove this temporary fix.